api/v1/models/Mentor.js

```diff
@@ -0,0 +1,43 @@
+var _ = require('lodash');
+var registration = require('../utils/registration');
+
+var Model = require('./Model');
+var MentorProjectIdea = require('./MentorProjectIdea');
+var Mentor = Model.extend({
+	tableName: 'mentors',
+	idAttribute: 'id',
+	validations: {
+		firstName: ['required', 'string', 'maxLength:255'],
+		lastName:  ['required', 'string', 'maxLength:255'],
+		shirtSize: ['required', 'string', registration.verifyTshirtSize],
+		github:     ['string', 'maxLength:50'],
```

Comment:
Yeah I think we should make it required. I'm going to leave it not-required in the schema just in case we need to add people that don't have githubs, but in the API we should mark it as required, yes.